src/com/sindhu/webapplication/ToDoList.java

```diff
@@ -24,43 +24,50 @@
         }
         return toDoListIds;
     }
+
     public static List<ToDoItem> toDoListItems(String username) {
         List<ToDoItem> toDoList = new ArrayList<>();
         try {
-            String query = "SELECT ID, toDoItem FROM To_Do_Lists WHERE username=" + "'" + username + "'";
+            String query = "SELECT ID, toDoItem, timer FROM To_Do_Lists WHERE username=" + "'" + username + "'";
             Connection connection = Database.getConnection();
-            if(connection == null){
+            if (connection == null) {
                 return toDoList;
             }
             Statement statement = connection.createStatement();
             ResultSet resultSet = statement.executeQuery(query);
             while (resultSet.next()) {
-                toDoList.add(new ToDoItem(resultSet.getLong(1), resultSet.getString(2)));
+                resultSet.getString("timer");
```

Comment:
store this into a variable called `timerValue`. and use it on line number 44. we dont have to call  `resultSet.getString("timer")` multiple times.